dashboard/modules/dashboard_sdk.py

```diff
@@ -188,7 +188,14 @@ def __init__(
         cluster_info = parse_cluster_info(
             address, create_cluster_if_needed, cookies, metadata, headers
         )
-        self._address = cluster_info.address
+
+        # Remove any trailing slashes
+        self._address = cluster_info.address.rstrip("/")
+        logger.debug(
+            "The submission address cannot contain trailing slashes. Removing "
+            f'them from the requested submission address of "{address}".'
+        )
+
```

Comment:
guard this with `if cluster_info.address.endswith("/")`?

Reply:
Good catch– I added a guard.